pkg/scalers/azure_log_analytics_scaler_test.go

```diff
@@ -219,27 +219,6 @@ type parseMetadataMetricNameTestData struct {
 	metricName  string
 }
 
-var testParseMetadataMetricName = []parseMetadataMetricNameTestData{
-	// WorkspaceID
-	{map[string]string{"tenantIdFromEnv": "d248da64-0e1e-4f79-b8c6-72ab7aa055eb", "clientIdFromEnv": "41826dd4-9e0a-4357-a5bd-a88ad771ea7d", "clientSecretFromEnv": "U6DtAX5r6RPZxd~l12Ri3X8J9urt5Q-xs", "workspaceIdFromEnv": "074dd9f8-c368-4220-9400-acb6e80fc325", "query": query, "threshold": "1900000000"}, 0, "azure-log-analytics-074dd9f8-c368-4220-9400-acb6e80fc325"},
-	// Custom Name
-	{map[string]string{"metricName": "testName", "tenantIdFromEnv": "d248da64-0e1e-4f79-b8c6-72ab7aa055eb", "clientIdFromEnv": "41826dd4-9e0a-4357-a5bd-a88ad771ea7d", "clientSecretFromEnv": "U6DtAX5r6RPZxd~l12Ri3X8J9urt5Q-xs", "workspaceIdFromEnv": "074dd9f8-c368-4220-9400-acb6e80fc325", "query": query, "threshold": "1900000000"}, 1, "azure-log-analytics-testName"},
-}
-
-func TestLogAnalyticsParseMetadataMetricName(t *testing.T) {
```

Comment:
I remove this chunk because it is testing against an object variable `metricName` which we don't use